e2e/test/helpers/logging/VerboseTestLogger.cs

```diff
@@ -9,7 +9,6 @@ public static class VerboseTestLogger
     {
         public static void WriteLine(string message)
         {
-            Console.WriteLine(message);
```

Comment:
If we don't log to console and event logging isn't enabled, will we get any logs?

Reply:
If event logging isn't enabled then we won't capture any logs => nothing would be written to console.